Documentation/dev-guide/apispec/swagger/rpc.swagger.json

```diff
@@ -8,6 +8,16 @@
     "http",
     "https"
   ],
+  "securityDefinitions" : {
```

Comment:
@heyitsanthony yes, please excuse my ignorance I am just learning how all of this tooling works together.  I will research proper solution further.